### PR TITLE
code-input Adding new languages

### PR DESCRIPTION
**Pull request by ajonp**
**Type of change (check at least one)**

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Documentation (fix or update to documentation)
- [ ]  Maintenance
- [ ]  Other, please describe:

**Does this change require a documentation update? (Check one)**

- [ ]  Yes
- [x]  No

**Current behavior**
[code-input](https://github.com/sanity-io/sanity/tree/next/packages/%40sanity/code-input) Adding new languages
Missing often used languages for webdev in the `code-input` plugin.

**Description**

Added these additional languages to the `code-input` plugin. Tested locally by building the repo and then moving the code-input package to my sanity studio locally. Also ran `npm run test-all`. In addition I alphabetized the list to make it easier in the future when languages are needed.

**Note for release**

Ace offers several languages https://ace.c9.io/. I use a lot more in my every day code samples and thought some of the more common ones should be added. 

**Checklist** 

- [x]  I have read the [Contributing Guidelines](https://github.com/sanity-io/sanity/blob/next/CONTRIBUTING.md)
- [x]  The PR title includes a link to the relevant issue
- [x]  The PR title is appropriately formatted: `[some-package] PR title (#123)`
- [x]  The code is linted
- [x]  The test suite is passing
`lerna success run Ran npm script 'test' in 25 packages in 60.9s:`
- [x]  Corresponding changes to the documentation have been made

### CODE REVIEW

File: packages/@sanity/code-input/src/config.js [hunk 1]
```diff
@@ -1,18 +1,26 @@
 export const SUPPORTED_LANGUAGES = [
   {title: 'Batch file', value: 'batchfile'},
   {title: 'CSS', value: 'css'},
-  {title: 'SASS', value: 'sass'},
-  {title: 'SCSS', value: 'scss'},
+  {title: 'CSarp', value: 'csharp'},
```
Comment (kristofferj) [inline packages/@sanity/code-input/src/config.js]: Small typo :)

Comment (kristofferj) [reply packages/@sanity/code-input/src/config.js]: Maybe C# is a good title?

Comment (skogsmaskin) [reply packages/@sanity/code-input/src/config.js]: This should be fixed before we merge.

**Comment by l5x** [comment]
@skogsmaskin can this be merged? ;-)

**Comment by mmgj** [comment]
Any chance of getting this through the goalpost any time soon? I really want my code inputs to support TypeScript.

**Comment by skogsmaskin** [comment]
Closing this, it's followed up in a new PR: https://github.com/sanity-io/sanity/pull/2676

**Pull request closed without merging**